docs/source/commands/peerlifecycle.md

```diff
@@ -375,6 +375,36 @@ Get installed chaincodes on peer:
 Package ID: myccv1:a7ca45a7cc85f1d89c905b775920361ed089a364e12a9b6d55ba75c965ddd6a9, Label: myccv1
 ```
 
+  * You can also use the `--output` flag to have the CLI format the output as
+    JSON.
+
+    ```
+    peer lifecycle chaincode queryinstalled --peerAddresses peer0.org1.example.com:7051 --output json
+    ```
+
+    If successful, the command will return the chaincodes you have installed as JSON.
+
+    ```
+    {
+            "installed_chaincodes": [
+                    {
+                            "package_id": "mycc_1:aab9981fa5649cfe25369fce7bb5086a69672a631e4f95c4af1b5198fe9f845b",
+                            "label": "mycc_1",
+                            "references": {
+                                    "mychannel": {
+                                            "chaincodes": [
+                                                    {
+                                                            "name": "mycc",
+                                                            "version": "1"
+                                                    }
+                                            ]
+                                    }
+                            }
+                    }
+            ]
+    }
+    ```
```

Comment:
Can we decrease the size of the tabs to two spaces, something that reduces that amount of whitespace

Reply:
Thank you for having reviews.
New patch decreates the size of the tabs to two spaces for their JSON blocks.